release.py

```diff
@@ -340,10 +340,6 @@ def select_gpg_key():
 
 cmd("Building artifacts", "gradle", cwd=kafka_dir, env=jdk7_env)
 cmd("Building artifacts", "./gradlew clean releaseTarGzAll aggregatedJavadoc", cwd=kafka_dir, env=jdk7_env)
-# This should be removed with KAFKA-4421
-cmd("Building artifacts for Scala 2.12", "./gradlew releaseTarGz -PscalaVersion=2.12", cwd=kafka_dir, env=jdk8_env)
-cmd("Copying artifacts", "cp %s/core/build/distributions/* %s" % (kafka_dir, artifacts_dir), shell=True)
-cmd("Copying artifacts", "cp -R %s/build/docs/javadoc %s" % (kafka_dir, artifacts_dir))
```

Comment:
Hmm, I don't think this is right. We still need to build Scala 2.12 separately since it requires Java 8.

Reply:
I thought KAFKA-4421 already manages this? If I misunderstand maybe I should remove this line?

Reply:
yeah, discussed quickly offline but the comment got outdated when ismael closed it due to the addition of this script :) removing just the comment is what we would want. looking forward to the day when we can stick to just jdk8...